lib/services/firewall.pm

```diff
@@ -0,0 +1,75 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Package for firewall service tests
+#
```

Comment:
I think we can add some description about the reboot_gnome issue depend on this transfer on s390x.

Reply:
added, please help me to check if it's correct.

Reply:
Seems ok, thanks.